docs/api-reference/create-next-app.md

```diff
@@ -45,9 +45,17 @@ Options:
 
     Initialize as a JavaScript project.
 
+  --tailwind
+
+    Initialize with Tailwind CSS config. (default)
+
+  --no-tailwind
+
+    Initialize without Tailwind CSS config.
```

Comment:
Decided to not add a separate flag like "--css-modules", but let me know.

Reply:
Does this mean the _default_ is Tailwind?

Reply:
Yes!